onyo/lib/tests/test_commands_set.py

```diff
@@ -466,82 +341,44 @@ def test_onyo_set_depth(inventory: Inventory) -> None:
     # assert inventory.repo.git.is_clean_worktree()
 
 
-@pytest.mark.repo_contents(
-    ["one_that_exists.test", "type: one\nmake: that\nmodel: exists\nserial: test\nsome_key: value"])
 @pytest.mark.ui({'yes': True})
-def test_onyo_set_depth_zero(inventory: Inventory) -> None:
-    """Calling `onyo_set(depth=0)` is legal and selects
-    all assets from all subpaths."""
-    key_value = {"this_key": "that_value"}
+def test_onyo_set_asset_dir(inventory: Inventory) -> None:
```

Comment:
Maybe this happens already and I just overlooked it (the diff is a bit weird to read at times).

There should be another explicit test that verifies that `onyo_set()` turns an asset into an asset dir.

Reply:
Lines 367ff and 377ff do that (both directions).